runtime/sema/variable_activations.go

```diff
@@ -298,15 +298,12 @@ func (a *VariableActivations) declare(declaration variableDeclaration) (variable
 			existingVariable.ActivationDepth == depth ||
 			existingVariable.ActivationDepth == 0) {
 
-		err = &RedeclarationError{
+		return nil, &RedeclarationError{
```

Comment:
Outside of the scope of changes in this PR but it popped into my eye - this function (`declare`) now doesn't use named return values and returns explicitly. The named return values can be removed - less things to keep in your head when reading the code.

Reply:
Good point! I removed the variable names in 2fa741bbbf78f7dfe07ac5e64650324e4c8d9650